src/static/styles/main.scss

```diff
@@ -204,6 +204,12 @@ h6,
   z-index: 2;
 }
 
+.warning-feedback {
+  @extend .invalid-feedback;
```

Comment:
Not sure i'll use the `extend` macro here.
the idea is nice, but the semantic is a bit different.

Reply:
I removed the @extend. The ruleset is really small, so the benefits of extending another class were not huge anyways